infra/gcp/README.md

```diff
@@ -0,0 +1,40 @@
+# Knative GCP Infrastructure
+
+We use Terraform to manage our Google Cloud infrastructure.
+
+- [knative-dns](./dns)
+
+  This is the knative-dns project. It holds the `knative.dev` and `knative.team` DNS Zones
+
+- [knative-gsuite](./gsuite)
+
+  This is the knative-gsuite project. It holds a service account that can access Google Workspace Admin API.
+
+- [knative-boskos-01…100](./boskos)
+
+  This the knative-boskos-XX project pools.
+
+
+# Bootstrapping Terraform - One Time Setup
+
+Terraform needs to be bootstrapped manually before it can be used. This process was done during Knative CNCF Infrastructure Migration. It is noted here for completeness and for potential troubleshooting.
+
+This needs to be ran by a person.
+
+```
+# Get the ORG_ID
+ORG_ID=$(gcloud organizations describe knative.team --format json | jq .name -r | sed 's:.*/::')
+
+#Create the knative-seed project
+
+gcloud projects create knative-seed --organization $ORG_ID --name "Knative Seed"
+
+#Create the terraform service account
+
+gcloud iam service-accounts create terraform —-display-name “Terraform” --project knative-seed
```

Comment:
What IAM bindings do we need for this service account? 
Also, is this one-off, or do we still need this service account and `knatve-seed` project for future operations?

Reply:
It is already been added.

```
 REDACTED  MCW0CDP3YY  ~  Desktop  Git  $  gcloud organizations get-iam-policy 22054930418
bindings:
- members:
  - domain:knative.team
  role: roles/billing.creator
- members:
  - serviceAccount:service-org-22054930418@security-center-api.iam.gserviceaccount.com
  role: roles/cloudfunctions.serviceAgent
- members:
  - domain:knative.team
  - group:kn-infra-gcp-org-admins@knative.team
  role: roles/cloudsupport.techSupportEditor
- members:
  - group:kn-infra-gcp-org-admins@knative.team
  role: roles/logging.privateLogViewer
- members:
  - domain:knative.team
  role: roles/logging.viewer
- members:
  - group:kn-infra-gcp-org-admins@knative.team
  - serviceAccount:terraform@knative-seed.iam.gserviceaccount.com
  role: roles/orgpolicy.policyAdmin
- members:
  - group:kn-infra-gcp-org-admins@knative.team
  - serviceAccount:terraform@knative-seed.iam.gserviceaccount.com
  role: roles/owner
- members:
  - group:kn-infra-gcp-org-admins@knative.team
  - serviceAccount:terraform@knative-seed.iam.gserviceaccount.com
  role: roles/resourcemanager.folderAdmin
- members:
  - group:kn-infra-gcp-org-admins@knative.team
  - serviceAccount:terraform@knative-seed.iam.gserviceaccount.com
  role: roles/resourcemanager.organizationAdmin
- members:
  - domain:knative.team
  - group:kn-infra-gcp-org-admins@knative.team
  - serviceAccount:terraform@knative-seed.iam.gserviceaccount.com
  role: roles/resourcemanager.projectCreator
- members:
  - serviceAccount:service-org-22054930418@security-center-api.iam.gserviceaccount.com
  role: roles/securitycenter.serviceAgent
- members:
  - serviceAccount:service-org-22054930418@security-center-api.iam.gserviceaccount.com
  role: roles/serviceusage.serviceUsageAdmin
etag: BwXbiEECEGA=
version: 1
```

We still need the service account as that is the account prow will use to do terraform changes. Humans will only need to run terraform if prow is unavailable.

knative-seed project holds the service account and the terraform state bucket